app/views/supervisor/dashboard/index.html.slim

```diff
@@ -1,11 +1,26 @@
 h2 Supervisor Dashboard
-p Alpha Version
+h5 Alpha Version
+
+div.dashboard-box
+  section.team-title
+    h3 Overview
+    h5 Under Construction
+  section.dashboard
+    p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
+    p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
+    /this is where all the aggregated messages and info should go
+    /= flash[:notice]
 
 div.clearfix
   - supervised_teams.each do |t|
     div.dashboard-box
       section.team-title
         h3 = link_to "Team " + t.name, t
+        div = render "#{ t.performance.score }"
+        - if t.performance.score == :orange
+          - flash.notice = "Team #{t.name} needs your attention"
```

Comment:
Hm, does that work with multiple teams having the same score? It looks as if it would overwrite stuff.